<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>606</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>606</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>606</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Crumby API Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="crumbs_list_view">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>521</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="crumby_description">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="crumby_recipe_name">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="crumby_recipe_description">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>481</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="reload_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ingredients_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Ingredients</string>
    </property>
   </widget>
   <widget class="QPushButton" name="instructions_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Instructions</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRecipes">
    <property name="title">
     <string>Recipes</string>
    </property>
    <addaction name="recipesActionOpen"/>
   </widget>
   <widget class="QMenu" name="menuIngredients">
    <property name="title">
     <string>Ingredients</string>
    </property>
    <addaction name="ingredientsActionOpen"/>
   </widget>
   <addaction name="menuRecipes"/>
   <addaction name="menuIngredients"/>
  </widget>
  <action name="recipesActionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="ingredientsActionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
